templates/group.hbs

```diff
@@ -9,7 +9,13 @@
 			{{{label}}}
 		</div>
 		<div class="group-facet-container-outer">
-			<div class="group-facet-container"></div>
+			{{#ifCond total '===' 0}}
+				<div class="group-facet-placeholder-container">
+					{{{groupPlaceholder}}}
```

Comment:
what happens if `groupPlaceholder` is undefined and the total is 0?

Reply:
It will not show anything, I can add a default value for it

Reply:
Its fine to not show anything.  Was more concerned about it throwing an exception or something.